_posts/2021-04-17-dd_00001_intro.md

```diff
@@ -0,0 +1,324 @@
+---
+layout: single
+title: Introduction to Digital Design
+category: Digital Design
+permalink: /categories/digital_design/intro
+excerpt: "A gentle introduction to the field of digital chip design"
+author: Kirthihan Yasotharan
+toc: true
+toc_label: "Content"
+toc_sticky: true
+toc_icon: "align-justify"
+#sidebar:
+#  nav: "getting_started_0"
+header:
+  overlay_color: "#000"
+  overlay_filter: "0.5"
+  overlay_image: /assets/images/digital_design.jpg
+---
+
+Welcome to our first post on **Digital Design**! We are glad that you found your
+way into this very exciting topic. In this post, we will try to give you an
+overview of the various aspects of digital design. For each topic, we will of
+course make a detailed and comprehensive post. This series intends to share our
+knowledge and experience in designing digital systems with you. Unlike many
+other resources, we will start from the ground up and explain the basic
+principles behind many of the technologies in digital design in a simple way.
+So, if you've never heard of digital design before, this is the right place to
+start! And, if you are already an experienced student or engineer, you are also
+welcome, of course; maybe you might learn something new along the way. We also
+invite you (newbies + experienced) to give us feedback and ideas for future
+posts!
+
+Now, let's jump in, shall we?
+
+If you look around you, you will probably find at least one _electronic_ device
+near you. In fact, you are probably reading this post through one of those
+devices. Have you ever wondered, how such small devices which surround us every
+day, can do so much powerful things? For most people, their devices are just
+made up of "black magic", and they "just work". But if you have made it until
+this point, you are probably interested in _what_ that "black magic" actually
+_is_, right? Then keep on reading and we will shed some light on this complex,
+but fascinating topic!
+
+## Microchips
+
+All of our intelligent devices are powered by what are called "**microchips**".
```

Comment:
Should we use "" or ' '. I used the single quote

Reply:
I removed the quotes here, as they are not necessary in this context (922e3bd).

But we can discuss which quotes to choose.